### DISCUSSION TITLE
Request a Refund

### DISCUSSION THREAD

**Discussion by Ethan-herne** [aclap-dev/video-downloadhelper#1089] [Help and Q&A]
### Your question

I'd like to get a refund, I bought the license with the hope of it making the plugin work for a specific website. I only wanted it for that website and it didn't work.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"9b7aa7bdeea0487d"}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/ethanherne/dwhelper"
        }
    },
    "concurrent_downloads_max": 4,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 27,
    "successfull_dl": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Just out of curiosity, what web site are you talking about?  If it's a site that requires a user registration, then never mind.  But if it's a public site that anybody could look at, I'd like to see if VDH for Firefox would work on it.

None of which is meant to get in the way of your refund, which of course I can't do anything about.

**Reply by SanTyaGooo** [reply]
Я качал видео с сайта, где требовалась регистрация. Но минимум 3 недели оттуда скачать не могу. На общедоступных работает

**Reply by Mattmjs** [reply]
@SanTyaGooo , what do you mean you can't download from there ? Is there no detection or is there a download failure.
If there is a failure you can submit a report about or provide some more information here. Click the the 3 dots button next
to download and look down to details. A  tab will open with more about the detected video. Copy this as text ( not an image )
and paste it here.

**Comment by jcfrog** [comment]
@Ethan-herne Hi, you have been refunded. Should be available in a few business days.
